drivers/i2s/i2s_ll_stm32.c

```diff
@@ -802,7 +842,13 @@ static void rx_stream_disable(struct stream *stream, const struct device *dev)
 	const struct i2s_stm32_cfg *cfg = dev->config;
 
 	LL_I2S_DisableDMAReq_RX(cfg->i2s);
+#ifdef CONFIG_SOC_SERIES_STM32H7X
```

Comment:
Most likely, additions/differences required for H7 support are similar to the ones that were needed on SPI, and will be needed as well for SoC with similar IPs (H7, wba, ..).
Would you mind adding a dedicated compatible "st,stm32h7-i2s" to deal with it ?
(See STM32 SPI as example)

Reply:
Sure! Please take a look, I think I got what you mean

Reply:
@erwango Please let me know if I should resolve, not sure how it works here

Reply:
That's globally correct!